From 774e33959dad5aa656de07a4bf4193828b02097b Mon Sep 17 00:00:00 2001
From: Christian Stewart <christian@paral.in>
Date: Tue, 14 Aug 2018 15:05:13 -0700
Subject: [PATCH] libfdt redefinition errors workaround

Patch made by:

$ find -name 'libfdt*.h' -exec sed -i 's/ _LIBFDT_/ LIBFDT_/g' {} +
$ find -name 'fdt.h' -exec sed -i 's/ _FDT_H/ FDT_H/g' {} +

Fixes libfdt redefinition errors.

Signed-off-by: Christian Stewart <christian@paral.in>
---
 include/fdt.h                | 6 +++---
 include/libfdt.h             | 6 +++---
 include/libfdt_env.h         | 6 +++---
 lib/libfdt/libfdt_internal.h | 6 +++---
 4 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/include/fdt.h b/include/fdt.h
index 526aedb515..74961f9026 100644
--- a/include/fdt.h
+++ b/include/fdt.h
@@ -1,5 +1,5 @@
-#ifndef _FDT_H
-#define _FDT_H
+#ifndef FDT_H
+#define FDT_H
 /*
  * libfdt - Flat Device Tree manipulation
  * Copyright (C) 2006 David Gibson, IBM Corporation.
@@ -108,4 +108,4 @@ struct fdt_property {
 #define FDT_V16_SIZE	FDT_V3_SIZE
 #define FDT_V17_SIZE	(FDT_V16_SIZE + sizeof(fdt32_t))
 
-#endif /* _FDT_H */
+#endif /* FDT_H */
diff --git a/include/libfdt.h b/include/libfdt.h
index e2bc2e00c1..29d2b1646f 100644
--- a/include/libfdt.h
+++ b/include/libfdt.h
@@ -1,5 +1,5 @@
-#ifndef _LIBFDT_H
-#define _LIBFDT_H
+#ifndef LIBFDT_H
+#define LIBFDT_H
 /*
  * libfdt - Flat Device Tree manipulation
  * Copyright (C) 2006 David Gibson, IBM Corporation.
@@ -2134,4 +2134,4 @@ int fdt_next_region(const void *fdt,
 int fdt_add_alias_regions(const void *fdt, struct fdt_region *region, int count,
 			  int max_regions, struct fdt_region_state *info);
 
-#endif /* _LIBFDT_H */
+#endif /* LIBFDT_H */
diff --git a/include/libfdt_env.h b/include/libfdt_env.h
index 6c6845f76c..1fbcd0ef83 100644
--- a/include/libfdt_env.h
+++ b/include/libfdt_env.h
@@ -6,8 +6,8 @@
  * SPDX-License-Identifier:	LGPL-2.1+
  */
 
-#ifndef _LIBFDT_ENV_H
-#define _LIBFDT_ENV_H
+#ifndef LIBFDT_ENV_H
+#define LIBFDT_ENV_H
 
 #include "compiler.h"
 #include "linux/types.h"
@@ -32,4 +32,4 @@ typedef __be64 fdt64_t;
 /* adding a ramdisk needs 0x44 bytes in version 2008.10 */
 #define FDT_RAMDISK_OVERHEAD	0x80
 
-#endif /* _LIBFDT_ENV_H */
+#endif /* LIBFDT_ENV_H */
diff --git a/lib/libfdt/libfdt_internal.h b/lib/libfdt/libfdt_internal.h
index 9a79fe85dd..8fdda7ee79 100644
--- a/lib/libfdt/libfdt_internal.h
+++ b/lib/libfdt/libfdt_internal.h
@@ -1,5 +1,5 @@
-#ifndef _LIBFDT_INTERNAL_H
-#define _LIBFDT_INTERNAL_H
+#ifndef LIBFDT_INTERNAL_H
+#define LIBFDT_INTERNAL_H
 /*
  * libfdt - Flat Device Tree manipulation
  * Copyright (C) 2006 David Gibson, IBM Corporation.
@@ -47,4 +47,4 @@ static inline struct fdt_reserve_entry *_fdt_mem_rsv_w(void *fdt, int n)
 
 #define FDT_SW_MAGIC		(~FDT_MAGIC)
 
-#endif /* _LIBFDT_INTERNAL_H */
+#endif /* LIBFDT_INTERNAL_H */
-- 
2.35.1

